Resources/Prototypes/Entities/Structures/Furniture/dresser.yml

```diff
@@ -51,6 +51,8 @@
   parent: Dresser
   suffix: Filled
   components:
+  - type: StaticPrice
+    price: 15
```

Comment:
Isn't this the price the test is saying is wrong?

Reply:
Oh goddamnit you're right.

Reply:
There you go. Fixed. :)